emgapi/third_party_metadata.py

```diff
@@ -141,3 +142,18 @@ def get_publication_annotations_existence_for_sample(sample, study_query_limit=8
         response[QUERY_POSSIBLE] = True
 
     return response
+
+
+def get_contextual_data_clearing_house_metadata(sample):
+    """
+    Fetch sample metadata from the Elixir Contextual Data Clearing House, if present.
+    :param sample: a MGnify Sample model instance.
+    :return: curations from the CDCH.
+    """
+    response = requests.get(f'{settings.ELIXIR_CDCH["sample_metadata_endpoint"]}{sample.accession}')
+    try:
+        assert response.status_code == 200
```

Comment:
Maybe it's better to do [response.raise_for_status()](https://docs.python-requests.org/en/master/_modules/requests/models/#Response.raise_for_status)?